src/Kfz24/Client/Aws/SqsClient.php

```diff
@@ -118,6 +147,43 @@ public function receive(array $args = [])
         return $result;
     }
 
+    /**
+     * @param array $args
+     *
+     * @return Promise
+     */
+    public function deleteMessageBatchAsync(array $args = []): Promise
```

Comment:
Can we here be more precise than `$args` ?

Reply:
I'm not sure how much we should change this. This is the same method signature that the AWS SDK uses. I could change `$args` to `$arguments` but it still is a collection of arguments and we can't really be more precise here.

Reply:
Okay, I get your point. Then let's leave it with args to stay the same as the SDK.